### DISCUSSION TITLE
Generates invalid spec if route has no leading slash

### DISCUSSION THREAD

**Discussion by chbndrhnns** [fastapi/fastapi#8377] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
def test_missing_leading_slash():
    app = fastapi.FastAPI()

    @app.get("model", response_model=None)
    def get():
        return None
```

### Description

* Copy the generated spec into editor.swagger.io

```yaml
openapi: 3.0.2
info:
  title: FastAPI
  version: 0.1.0
paths:
  model:
    get:
      summary: Get
      operationId: getmodel_get
      responses:
        '200':
          description: Successful Response
          content:
            application/json:
              schema: {}

```

* See problem:

```
Structural error at paths
should only have path names that start with `/`
Jump to line 5
```

### Operating System

macOS

### FastAPI Version

0.80

### Python Version

3.9.9

**Comment by JarroVGIT** [comment]
Interesting, I actually expected an error that you are not allowed to have paths without a leading `/`. It is implied when adding a router, where if you add a trailing slash you will get the message `A path prefix must not end with '/', as the routes will start with '/'`. 

All documentation is also implying (but nowhere asserting) that paths are with leading slashes. Might require an update of the docs, unless I am overlooking something.

**Comment by imacat** [comment]
Is there any reason you don't want to add the leading slash?
